doc/modules/svm.rst

```diff
@@ -599,57 +632,68 @@ Its dual is
    \textrm {subject to } & y^T \alpha = 0\\
    & 0 \leq \alpha_i \leq C, i=1, ..., n
 
-where :math:`e` is the vector of all ones, :math:`C > 0` is the upper bound,
-:math:`Q` is an :math:`n` by :math:`n` positive semidefinite matrix,
+where :math:`e` is the vector of all ones,
+and :math:`Q` is an :math:`n` by :math:`n` positive semidefinite matrix,
 :math:`Q_{ij} \equiv y_i y_j K(x_i, x_j)`, where :math:`K(x_i, x_j) = \phi (x_i)^T \phi (x_j)`
-is the kernel. Here training vectors are implicitly mapped into a higher
-(maybe infinite) dimensional space by the function :math:`\phi`.
+is the kernel. The terms :math:`\alpha_i` are called the dual coefficients.
+This dual representation highlights the fact that training vectors are
+implicitly mapped into a higher (maybe infinite)
+dimensional space by the function :math:`\phi`.
```

Comment:
It's really tricky to explain why the dual highlights this fact. Hmm

Reply:
"tricky"

I see what you did there.


(all jokes aside yeah, we can't really explain that unless we explain the kernel trick. I'll add a link to https://en.wikipedia.org/wiki/Kernel_method which is reasonably good)